doc/app/SOCKET_AT_commands.rst

```diff
@@ -898,7 +898,9 @@ Syntax
 
 ::
 
-   #XSEND=<handle>,<mode>,<flags>[,<data>]
+   #XSEND=<handle>,<mode>,<flags>,<data> for string and hex string modes
```

Comment:
We need to think how to structure the syntax. I don't recall that we would have different parameters in other commands.

Option 1: Keep it as is written now

Option 2:
```
   #XSEND=<handle>,<mode>,<flags>,<data> when ``<mode>`` is ``0`` or ``1``
```

Option 3:
```
   #XSEND=<handle>,<mode>,<flags>[,<data>|<data_len>]
```

Option 4:
```
   When ``<mode>`` is ``0`` or ``1``:
   #XSEND=<handle>,<mode>,<flags>,<data>
```

Feel free to vote :smile: